packages/framework/build.xml

```diff
@@ -1077,25 +1070,72 @@
                 </exec>
             </try>
             <finally>
-                <if>
-                    <not>
-                        <equals arg1="${path.env.test.existed}" arg2="true"/>
-                    </not>
-                    <then>
-                        <delete file="${path.root}/TEST"/>
-                    </then>
-                </if>
+                <phingcall target="tests-acceptance-after"/>
             </finally>
         </trycatch>
     </target>
 
+    <target name="tests-acceptance-after" description="Actions necessary to run after acceptance tests." hidden="true">
+        <available file="${path.env.acc}" type="file" property="path.env.acc.existed"/>
+        <if>
+            <equals arg1="${path.env.acc.existed}" arg2="true"/>
+            <then>
+                <delete file="${path.root}/ACCEPTANCE"/>
+            </then>
+        </if>
+        <phingcall target="domains-urls-replace"/>
```

Comment:
Isn't necessary to export data to elasticsearch too?

Reply:
Actually even that domains-urls-replace is not necessary as urls are generated on the fly

2086e6845470d24b6bc4f8a5d9c12a568fd9f09b